components/emails/contactEmailTemplate.jsx

```diff
@@ -0,0 +1,159 @@
+import {
+  Box,
+  Container,
+  Heading,
+  Text,
+  VStack,
+  HStack,
+  Separator,
+  CardBody,
+  Badge
+} from '@chakra-ui/react'
```

Comment:
i don't think we're going to be able to use chakra here without some hacking. i couldn't figure it out in the 10 mins i gave to this.

you need a chakra provider, and i believe that Resend is rendering this component outside of the chakra provider. 

i'll dm you the resend API key so you can test locally